heidelberg-university-faculty-of-medicine.csl

```diff
@@ -0,0 +1,323 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style xmlns="http://purl.org/net/xbiblio/csl" class="in-text" version="1.0" page-range-format="expanded" demote-non-dropping-particle="never">
```

Comment:
I also have been adding an XML comment to polyglot styles, so it's clear later why certain styles don't have a "default-locale". See https://github.com/citation-style-language/styles/search?utf8=%E2%9C%93&q=polyglot (an example is `<!-- Polyglot; used both with German and English -->`)